wasm/src/record/record_plaintext.rs

```diff
@@ -43,6 +46,16 @@ impl RecordPlaintext {
     pub fn gates(&self) -> u64 {
         ***self.0.gates()
     }
+
```

Comment:
Also always try to comment public `functions` `methods` and `structs` as they're meant to be part of the outward facing api. `Rustdoc` comments are specified with three slashes like `///` whereas regular comments which won't show up in the documation are two slashes `//`

```suggestion
/// Attempt to get the serial number of a record to determine whether or not is has been spent
```

Reply:
Great suggestion, this will make it easier for wasm-clients (like me!)